plugins/modules/redhat_manifest.py

```diff
@@ -25,10 +25,6 @@
     - Download and modify a Red Hat Satellite Subscription Manifest
 author:
     - "Sean O'Keeffe (@sean797)"
-requirements:
-    - "nailgun >= 0.28.0"
```

Comment:
:laughing:

Reply:
Yeah, I have NO idea how that has sneaked in ;)